tfx/components/statistics_gen/executor.py

```diff
@@ -89,12 +94,11 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     self._log_startup(input_dict, output_dict, exec_properties)
 
     stats_options = options.StatsOptions()
-    if STATS_OPTIONS_JSON_KEY in exec_properties:
+    if exec_properties.get(STATS_OPTIONS_JSON_KEY):
       stats_options_json = exec_properties[STATS_OPTIONS_JSON_KEY]
-      if stats_options_json:
-        # TODO(b/150802589): Move jsonable interface to tfx_bsl and use
-        # json_utils
-        stats_options = options.StatsOptions.from_json(stats_options_json)
+      # TODO(b/150802589): Move jsonable interface to tfx_bsl and use
+      # json_utils
+      stats_options = options.StatsOptions.from_json(stats_options_json)
```

Comment:
since you changed it.. make it even concise and avoid double lookups?

```
stats_options_json = exec_properties.get(STATS_OPTIONS_JSON_KEY)
if stats_options_json is not None:
  ...
``

Reply:
yes, i think this is more concise, thanks for the catch